centos_ci.sh

```diff
@@ -208,6 +216,52 @@ function docs_tar_upload() {
   echo "Find docs tar here http://artifacts.ci.centos.org/minishift/minishift/docs/$LATEST."
 }
 
+function bump_to_release_version() {
+  # Set Terminal
+  export TERM=xterm-256color
+  # Add git a/c identity
+  git config user.email "admn.minishift@gmail.com"
+  git config user.name "Minishift Bot"
+  # Export GITHUB_ACCESS_TOKEN
+  export GITHUB_ACCESS_TOKEN=$GITHUB_TOKEN
+  # Create master branch as git clone in CI doesn't create it
+  git checkout -b master
+  # Bump version and commit
+  sed -i "s|MINISHIFT_VERSION = .*|MINISHIFT_VERSION = $RELEASE_VERSION|" Makefile
+  git add Makefile
+  git commit -m "cut v$RELEASE_VERSION"
+  # Push update
+  # TODO: Change to minishift once final review is done
+  git push https://$1:$GITHUB_TOKEN@github.com/$1/minishift master
+}
+
+function add_release_notes() {
+  # TODO: Change to minishift once final review is done
+  id=$(curl -s "https://api.github.com/repos/${1}/minishift/releases" | jq --arg release $RELEASE_VERSION -r ' .[] | select(.name | test([$release])) | .id')
+
+  if [[ "$id" != "" ]]; then
+    # Generate required json payload for release note
+    ./scripts/release/issue-list.sh -r minishift -m $MILESTONE_ID | jq -Rn 'inputs + "\n"' | jq -s '{body:  add }' > json_payload.json
+    # Add release notes
+    # TODO: Change to minishift once final review is done
+    curl -H "Content-Type: application/json" -H "Authorization: token $GITHUB_TOKEN" \
+         --data @json_payload.json https://api.github.com/repos/${1}/minishift/releases/$id
+
+    # TODO: Change to minishift once final review is done
+    echo "Minishift v$RELEASE_VERSION has been successfully released. Find the latest release here https://github.com/${1}/minishift/releases/tag/v$RELEASE_VERSION."
+  else
+    echo "Couldn't find the release ID. Need to manually update the release note. Go to https://github.com/${1}/minishift/releases"
+  fi
+}
+
+function triger_docs_build() {
```

Comment:
Actually there is an alternative to a multi step build. We can just make the docs tarball build part of the release steps right? After all the methods are part of the script, right? So instead of triggering the docs build, we execute the required steps directly as part of the release job. Would that be easier?